app/src/main/res/values/styles_v2.xml

```diff
@@ -658,4 +654,18 @@
     <item name="android:paddingStart">@dimen/spacing_16</item>
     <item name="android:paddingTop">@dimen/spacing_16</item>
   </style>
+
+  <style name="Clinic.V2.TextAppearance.Identifier" />
```

Comment:
I can see that this is a bold style.
Can we stick to the name used in Zeplin designs - > `Clinic.V2.TextAppearance.Numeric-Bold`

Reply:
We don't need another `parent` style for `Clinic.V2.TextAppearance.Numeric-Bold`.
its parent is already `Clinic.V2.TextAppearance`.
So you can merge this with `Clinic.V2.TextAppearance.Identifier.Numeric` and rename it.

Reply:
I have pulled in the new styles from Zeplin and amended the commit.